Gemfile

```diff
@@ -24,8 +24,8 @@ platforms :ruby do
 end
 
 gem "puppet", :path => File.dirname(__FILE__), :require => false
-gem "facter", *location_for(ENV['FACTER_LOCATION'] || ['> 1.6', '< 3'])
-gem "hiera", *location_for(ENV['HIERA_LOCATION'] || '~> 1.0')
+gem "facter", *location_for(ENV['FACTER_LOCATION'] || ['> 2.0', '< 3'])
```

Comment:
Oh right, totally forgot about this change. Thanks!

I think facter should be `'< 4'` so that puppet 4 can be run against facter 3.